js/mexc3.js

```diff
@@ -1769,6 +1786,12 @@ module.exports = class mexc3 extends Exchange {
                 throw new ArgumentsRequired (this.id + ' createSwapOrder() requires a leverage parameter for isolated margin orders');
             }
         }
+        const reduceOnly = this.safeValue (params, 'reduceOnly', false);
+        if (reduceOnly) {
```

Comment:
@Dan-krm did you manage to place a swap order? I'm always getting a "contract not allowed" error regardless of the market